dwi_ml/data/hdf5_creation.py

```diff
@@ -570,27 +570,30 @@ def _process_one_streamline_group(
             if tmp_bundle_name.endswith('/ALL'):
                 bundles_dir = tmp_bundle_name.split('/ALL')
                 bundles_dir = ''.join(bundles_dir[:-1])
-                bundles2 = [str(p) for p in subj_dir.glob(bundles_dir + '/*')]
+                bundles_sublist = [
+                    tmp_bundle_name.replace('/ALL', '/' + os.path.basename(p))
+                    for p in subj_dir.glob(bundles_dir + '/*')]
             else:
-                bundles2 = [tmp_bundle_name]
+                bundles_sublist = [tmp_bundle_name]
 
             # Either a loop on "ALL" or a loop on only one file,
             # tmp_bundle_name.
-            for bundle_name in bundles2:
+            for bundle_name in bundles_sublist:
                 bundle_name = bundle_name.replace('*', subj_id)
                 bundle_file = subj_dir.joinpath(bundle_name)
```

Comment:
There is still the problem that `bundle_name` is actually the bundle's path, even when not using `ALL`

Reply:
oh, I just tested the 'all' case.

Reply:
I'm not sure I see the issue. In the 'all' case, I fixed it with at the new line 514. In the other case, tmp_bundle_name is directly read from the config file, it shouldn't be the whole path.